hub/api/dataset.py

```diff
@@ -227,10 +235,94 @@ def like(
     @staticmethod
     @hub_reporter.record_call
     def ingest(
-        path: str, src: str, src_creds: dict, overwrite: bool = False
+        src: str, dest: str, src_creds: dict, compression: str, overwrite: bool = False
```

Comment:
I would prefer to have path as the first argument for the sake of uniformity

Reply:
path doesn't make sense here because it's not a hub dataset. it is source because it's an unstructured dataset

Reply:
```suggestion
        src: str, dest: str, dest_creds: dict, compression: str, overwrite: bool = False
```

Reply:
We need both src_creds and dest_creds as optional inputs.